tests/functional/aws-node-sdk/test/object/abortMPU.js

```diff
@@ -45,23 +48,24 @@ describe('Abort MPU', () => {
                 Key: key,
                 UploadId: uploadId,
             }).promise()
-            .then(() => bucketUtil.empty(bucket))
-            .then(() => bucketUtil.deleteOne(bucket))
+                .then(() => bucketUtil.empty(bucket))
+                .then(() => bucketUtil.deleteOne(bucket))
         );
 
         // aws-sdk now (v2.363.0) returns 'UriParameterError' error
         // this test was not replaced in any other suite
         it.skip('should return InvalidRequest error if aborting without key',
-        done => {
-            s3.abortMultipartUpload({
-                Bucket: bucket,
-                Key: '',
-                UploadId: uploadId },
-            err => {
-                checkError(err, 'InvalidRequest', 'A key must be specified');
-                done();
+            done => {
+                s3.abortMultipartUpload({
+                    Bucket: bucket,
+                    Key: '',
+                    UploadId: uploadId
+                },
+                    err => {
+                        checkError(err, 'InvalidRequest', 'A key must be specified');
+                        done();
+                    });
```

Comment:
better like this?
```suggestion
                }, err => {
                    checkError(err, 'InvalidRequest', 'A key must be specified');
                    done();
                });
```

(same at line 277, if you agree)